test-data/unit/check-enum.test

```diff
@@ -1881,4 +1881,8 @@ class C(IntEnum):
 def f1(c: C) -> None:
     x = {'x': c.value}
     reveal_type(x)  # N: Revealed type is "builtins.dict[builtins.str*, builtins.int]"
+
+def f2(c: C, a: Any) -> None:
+    x = {'x': c.value, 'y': a}
```

Comment:
Will `y = {'y': a, 'x': c.value}` be helpful here as well?

Reply:
That's a good idea, I'll add it here.